frontend/src/concepts/notebooks/StartNotebookModal.tsx

```diff
@@ -193,25 +186,12 @@ const StartNotebookModal: React.FC<StartNotebookModalProps> = ({
   };
 
   const renderLogs = () => (
-    <Panel style={{ overflowY: 'auto', height: '100%', padding: SmallSpacerSize.var }}>
+    <Panel className="odh-modal__scrollable-panel">
```

Comment:
Would it be simpler for us to use this: https://www.patternfly.org/components/panel/#panel the `isScrollable` prop?

Reply:
sure, updated